tests/v1/tpu/test_sampler.py

```diff
@@ -51,9 +51,10 @@ def test_sampler_different(model_name: str):
                 min_p=0.8,
                 max_tokens=64,
                 # Vary number of ks
-                top_k=random.randint(4, 12)) for _ in range(B)
+                top_k=random.randint(4, 12),
+                top_p=random.random()) for _ in range(B)
```

Comment:
Is close to 0 valid for top p?

Reply:
yeah it's the smallest set with sum of probs > p